src/style.scss

```diff
@@ -107,6 +107,11 @@ slot {
   user-select: none;
   white-space: nowrap;
   width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: var(--auro-size-xxs);
```

Comment:
By making this selector `display:flex` the element now appears as a `block` element. Buttons by default are `inline-block` elements. I do not see changing the display type of an element as a reasonable outcome for gaining space between text and an icon used as content. 

You can create a selector that will target `auro-icon` directly and add a margin as needed. This will not have a negative effect on the display type of the element. 

Also, I would argue that this is not a feature add, but a bug correction.

Reply:
FYI... it is super helpful, especially with visual changes to have a `before` and `after` screenshot for reference when reviewing the PR. 

This also applied to the issues themselves. Giving the reviewer visual context of the pending change is super helpful.

Reply:
The downside to the margin approach is the we have to know whether the icon is to the left or right of the text.  

That's a good call out on the inline-block thing. Would inline-flex behave similar to inline-block?